index.html

```diff
@@ -90,21 +90,86 @@ <h2>Work</h2>
                 <li><i class="fa-solid fa-m"></i></li>
             </ul>
         </div>
-        <article class="work">
+        <section class="work">
             <span class="arrow"><i class="fa-solid fa-arrow-down-long"></i></span>
             <h2>Work</h2>
 
-        </article>
+        </section>
        </div>
         <div id="append" class="work-section">
-          <article class="top">
+          <section class="top">
             <div class="head">
                 <h2>MY RECENT WORK</h2>
             </div>
-          </article>
-         
-        </div>   
-    </main>
+          </section>
+
+          <section class="down">
+            <div class="box-1">
+                <img src="./images/ImgPlaceholder.png" alt="Blank square">
+                <div class="box-2">
+                    <h2>Multi-Post Stories</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <ul>
+                        <li>css</li>
+                        <li>Html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+     
+                    </ul>
+                    <button type="button" class="button">See Project</button>
+                 </div>
  
+            </div>
+            <div class="box-1 row__rev">
+             <img src="./images/ImgPlaceholder.png" alt="Blank square">
+             <div class="box-2">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul>
+                    <li>css</li>
+                    <li>Html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+ 
+                </ul>
+                <button type="button" class="button">See Project</button>

```

Comment:
Please align in order the button tag. in order to make your codes more readable and clean.